core/src/main/java/org/kiwix/kiwixmobile/core/utils/StyleUtils.kt

```diff
@@ -47,24 +43,13 @@ object StyleUtils {
     return Xml.asAttributeSet(parser)
   }
 
-  fun highlightUrl(text: String, url: String): Spanned {
-    return fromHtml(
-      text.replace(
-        url.toRegex(),
-        "<u><font color='blue'>$url</font></u>"
-      )
-    )
-  }
+  fun highlightUrl(text: String, url: String): Spanned =
```

Comment:
Can be changed to extension function

Reply:
@abdulwd I was about to ask for the same. I can't see the usage of this function. Should I delete it?

Reply:
Of course.